samples/subsys/shell/adc/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _shell-adc-sample:
+
+Analog to Digital Converter (ADC) shell example
+###############################################
+
+Overview
+********
+
+This example provides shell access to aquiring ADC samples
+
+Requirements
+************
+
+A board with ADC support and console
```

Comment:
I think that `console` is not needed for shell to be running.

Reply:
Fixed, thank you.